service/kernel/oneapi/service_defines_oneapi.h

```diff
@@ -32,5 +32,14 @@
             return return_obj;                                                \
         }                                                                     \
     }
+#define DAAL_ADD_STATUS_RETURN_IF_FAIL(status, status_ptr, return_obj)        \
```

Comment:
Why you use two statuses? Is it possible to use one status for check and use existing macro?